### DISCUSSION TITLE
智慧醫療-紀錄資料🏥

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#80] [General]
**新聞大綱：**
疫情期間，短時間內湧入大量篩檢民眾，增加問診時病歷缺漏或誤植風險，且防疫門診人員穿著防護衣手套不便打字輸入，為減輕醫護壓力並確保醫療品質，高雄長庚醫院與台灣人工智慧實驗室（Taiwan AI Labs）合作，開發AI零接觸智能輸入系統。
以口述輸入方式，運用語音辨識技術即時紀錄患者臨床症狀、病史，並串接醫院病歷系統與傳染病通報系統，目前已成功導入高雄長庚醫院急診室與篩檢站使用，未來將推展至長庚體系其他院區。
未來將進一步訓練AI自然語音辨識並理解醫病問診內容，自動取得重要資訊摘錄電子病歷，同時利用相關資訊協助診斷與風險分級，提升病歷繕寫品質與臨床照護安全。
高雄市政府今年以來籌組「高雄聯合學習智慧醫療聯盟」，推動市內四大醫學中心與本實驗室共同催生AI聯合學習合作框架，透過AI精準健康成果協作共享，帶動高雄智慧醫療產業發展與AI臨床落地應用。
**新聞心得：**
我覺得把語音輸入應用在看診真的很方便，可以縮短患者跟醫生確認資訊的時間，也讓醫護人員不會因為病患人數多而忙到輸入錯資料，未來進一步發展的自動摘錄成電子病歷對於整體診斷有更大的幫助，就感覺有點像大英雄天團裡的杯麵嗎，醫療小助手?!
**新聞來源：**
https://udn.com/news/story/7241/5830894
![image](https://user-images.githubusercontent.com/90883526/138580709-27143bcd-39d4-4fa9-ba24-8417376776bc.png)

**Comment by chia-kai-chang** [comment]
這邊有影片唷：https://youtu.be/6x923Q_je0c

下次上課會教大家使用**語音轉文字**

**Reply by boson13579** [reply]
我覺得影片中讓患著說一次，醫生再說一次感覺很讓費時間，反而會拖到下一位病患的時間，為何不在患者那邊也裝一個麥克風就好了呢?

**Comment by bww0401** [comment]
這樣真的能夠讓看診更有效率，也減少病毒接觸風險，在未來如果再加上能夠自動幫患者分類別的功能，更能夠讓醫生在看診時快速了解病人的嚴重性，可以視情況安排順序，病人也不會因此錯過黃金期。
